packages/core/src/button/inline-button.element.ts

```diff
@@ -50,14 +49,6 @@ export class CdsInlineButton extends CdsBaseButton {
   }
 
   static get styles() {
-    return [baseStyles, baseButtonStyles, styles];
-  }
-}
-
-registerElementSafely('cds-inline-button', CdsInlineButton);
-
-declare global {
-  interface HTMLElementTagNameMap {
-    'cds-inline-button': CdsInlineButton;
```

Comment:
The button was not registering in the prod build of storybook since it was not in the register.ts file